src/index.html

```diff
@@ -7,12 +8,68 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The SCSS file `index.scss` is linked here. Ensure that SCSS variables are placed in a separate file, and separate BEM block files are used for `card` and `stars`, as per the task requirements.